tests/acceptance/features/apiSharingNg/sharedWithMe.feature

```diff
@@ -3539,3 +3539,891 @@ Feature: an user gets the resources shared to them
+                          "type": "string",
+                          "pattern": "^%permissions_id_pattern%$"
+                        },
+                        "grantedToV2": {
+                          "type": "object",
+                          "required": ["user"],
+                          "properties": {
+                            "user": {
+                              "type": "object",
+                              "properties": {
+                                "displayName": {
+                                  "const": "Brian Murphy"
+                                },
+                                "id": {
+                                  "type": "string",
+                                  "pattern":"^%user_id_pattern%$"
+                                }
+                              },
+                              "required": [
+                                "displayName",
+                                "id"
+                              ]
+                            }
+                          }
+                        },
+                        "invitation": {
+                          "type": "object",
+                          "required": ["invitedBy"],
+                          "properties": {
+                            "invitedBy": {
+                              "type": "object",
+                              "required": ["user"],
+                              "properties": {
+                                "user": {
+                                  "type": "object",
+                                  "required": [
+                                    "displayName",
+                                    "id"
+                                  ],
+                                  "properties": {
+                                    "displayName": {
+                                      "const": ""
+                                    },
+                                    "id": {
+                                      "type": "string",
+                                      "pattern": "^%user_id_pattern%$"
+                                    }
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        },
+                        "roles": {
+                          "type": "array",
+                          "maxItems": 1,
+                          "minItems": 1,
+                          "items": {
+                            "type": "string",
+                            "pattern": "^%role_id_pattern%$"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    """
+
+
+  Scenario: user lists the file shared with them from personal space after the sharer is deleted
```

Comment:
Please, add scenarios per item from https://github.com/owncloud/ocis/issues/7826 and the PR title would be meaningful.